bool servoState
---
int32 result
string strRt